vienna/index.html

```diff
@@ -10,22 +10,27 @@
     <img src="./img/pyladies_logo.png" width="300px" />
     <h3>Hello Vienna!</h3>
     <p>PyLadies as an International movement aims to provide a friendly support network for women and a bridge to the larger Python world. Anyone with an interest in Python is encouraged to participate (yes, that means guys too)!</p>
-    <p>Our local chapter is planning a <a href="http://www.meetup.com/PyLadies-Vienna/events/130539712/">second workshop, September 7th</a>. That means: more (and different) Python tutorials and... more pizza! New to this group? We'll make sure all talks and tutorials are tailored at beginners, the second-time attendees will just go through the introduction a bit faster. But more on this later. Promised.</p>
-    <p>What you're getting in to? Well... Check out some <a href="http://www.flickr.com/photos/floordrees/sets/72157634723504650/">pictures</a> from our first event in July - sponsored by <a href="https://usersnap.com/">Usersnap</a> and <a href="https://www.checkio.org/">CheckiO</a> - and feel free to look at <a href="https://github.com/FloorD/pyladies_vienna">the slides of the lightning talks</a>! Like to get started by learning about the terminal and interactive Python (Mac users only)? <a href="https://github.com/FloorD/terminalmatrix">This way please</a>!</p>
+    <p>Our local chapter is planning the next workshop on <a href="http://www.meetup.com/PyLadies-Vienna/events/219911034">February 22nd</a>. This time we'll leave Django behind and have a little introduction to Flask, directly from its maintainer! How awesome is that? If you're new to the group, worry not: we are always happy about newcomers, and we always have coaches ready to help with python and installation basics.</p>
+    <p>What you're getting into? Well... Check out some <a href="http://www.flickr.com/photos/floordrees/sets/72157634723504650/">pictures</a> from our very first event in July 2013 - sponsored by <a href="https://usersnap.com/">Usersnap</a> and <a href="https://www.checkio.org/">CheckiO</a> - and feel free to look at <a href="https://github.com/FloorD/pyladies_vienna">the slides of the lightning talks</a>! Like to get started by learning about the terminal and interactive Python (Mac users only)? <a href="https://github.com/FloorD/terminalmatrix">This way please</a>!</p>
     
     <div id="button">
-      <span><a href="http://www.meetup.com/PyLadies-Vienna/events/130539712/">rsvp here</a></span>
+        <span><a href="http://www.meetup.com/PyLadies-Vienna/events/219911034">rsvp here</a></span>
     </div>
     
+    <!--
```

Comment:
@FloorD @alicetragedy Did you mean to comment out this whole block? If so, I would recommend just deleting the block of code.